docs/application/tizen-studio/release-notes/2-3-release-notes.md

```diff
@@ -0,0 +1,84 @@
+# Tizen Studio 2.3 Release Notes
+
+
+## IDE and Tools
+
+
+### New Features
+
+-   Tizen Studio
+	-   [JAVA 9](https://www.oracle.com/java/java9.html) support is added in Tizen Studio. Users can now download and install Tizen Studio along with JAVA 9.
+-   Base IDE
+	-   Tizen Studio is updated from [Eclipse Mars](https://www.eclipse.org/mars/) to [Eclipse Oxygen](https://projects.eclipse.org/releases/oxygen), to help user with improved performance and bug fixes.
+	-   CDT (C/C++ Development Tooling) is updated to 9.3.2 and the Debugger is migrated from CDI to DSF enabling faster debugging and easier extension.
+-	Emulator Manager
+	-   The Emulator Manager is updated to support JAVA 9.
+	-   [Intel&reg; HAXM](../setup/hardware-accelerated-execution-manager.md) is upgraded to 7.0.0 to support Tizen Emulators on Windows 10 and Mac.
+-	Device Manager
+	-   The Device Manager is updated to support JAVA 9.
+-	Profiling Tools
+	-   Dynamic Analyzer and TTrace are updated to support JAVA 9.
+-	Installer, Package Manager, and Uninstaller
+	-   Package Manager and Installer are updated to support JAVA 9.
+-	RT IDE (Integrated Development Environment)
+	-   Base IDE of RT IDE is migrated from Eclipse Mars to Eclipse Oxygen, to help user with improved performance and bug fixes.
+### Changed Features
+-	Certificate Manager
+	-   ST (Smart Things) certification is simplified by enabling auto certificate generation using Samsung Account sign-in.
```

Comment:
Hello, karthikthecool.
Before we merge this PR, I think we need one more fix to add a notice for 32bit host at here.
Just before the `Fixed Bugs` section, would you please add below note?

```
> **Note**
> From Tizen Studio 2.4 onwards, support for 32 bit host for Windows and Ubuntu will be removed.
```
Regards
Catherine